### ISSUE TITLE
SQLAlchemy issues warning for backref / back_populates

### ISSUE THREAD

**Issue by Glandos** [spiral-project/ihatemoney#630]
Currently, I have:
```
./Dev/ihatemoney/.venv/lib/python3.8/site-packages/sqlalchemy/orm/relationships.py:1993: SAWarning: Setting backref / back_populates on relationship Project.versions to refer to viewonly relationship ProjectVersion.version_parent should include sync_backref=False set on the Project.versions relationship.  (this warning may be suppressed after 10 occurrences)
  util.warn_limited(
./Dev/ihatemoney/.venv/lib/python3.8/site-packages/sqlalchemy/orm/relationships.py:1993: SAWarning: Setting backref / back_populates on relationship ProjectVersion.version_parent to refer to viewonly relationship Project.versions should include sync_backref=False set on the ProjectVersion.version_parent relationship.  (this warning may be suppressed after 10 occurrences)
  util.warn_limited(
./Dev/ihatemoney/.venv/lib/python3.8/site-packages/sqlalchemy/orm/relationships.py:1993: SAWarning: Setting backref / back_populates on relationship Person.versions to refer to viewonly relationship PersonVersion.version_parent should include sync_backref=False set on the Person.versions relationship.  (this warning may be suppressed after 10 occurrences)
  util.warn_limited(
./Dev/ihatemoney/.venv/lib/python3.8/site-packages/sqlalchemy/orm/relationships.py:1993: SAWarning: Setting backref / back_populates on relationship PersonVersion.version_parent to refer to viewonly relationship Person.versions should include sync_backref=False set on the PersonVersion.version_parent relationship.  (this warning may be suppressed after 10 occurrences)
  util.warn_limited(
./Dev/ihatemoney/.venv/lib/python3.8/site-packages/sqlalchemy/orm/relationships.py:1993: SAWarning: Setting backref / back_populates on relationship Bill.versions to refer to viewonly relationship BillVersion.version_parent should include sync_backref=False set on the Bill.versions relationship.  (this warning may be suppressed after 10 occurrences)
  util.warn_limited(
./Dev/ihatemoney/.venv/lib/python3.8/site-packages/sqlalchemy/orm/relationships.py:1993: SAWarning: Setting backref / back_populates on relationship BillVersion.version_parent to refer to viewonly relationship Bill.versions should include sync_backref=False set on the BillVersion.version_parent relationship.  (this warning may be suppressed after 10 occurrences)
  util.warn_limited(
```

The warning gives an advice, but I'd like some other comments on it.

**Comment by zorun** [comment]
I have a slightly different warning (it's the same issue though):

```
sqlalchemy/orm/relationships.py:1912: SAWarning: Setting backref / back_populates on relationship ProjectVersion.version_parent to refer to viewonly relationship Project.versions will be deprecated in SQLAlchemy 1.4, and will be disallowed in a future release.  viewonly relationships should not be mutated (this warning may be suppressed after 10 occurrences)
sqlalchemy/orm/relationships.py:1912: SAWarning: Setting backref / back_populates on relationship Project.versions to refer to viewonly relationship ProjectVersion.version_parent will be deprecated in SQLAlchemy 1.4, and will be disallowed in a future release.  viewonly relationships should not be mutated (this warning may be suppressed after 10 occurrences)
sqlalchemy/orm/relationships.py:1912: SAWarning: Setting backref / back_populates on relationship PersonVersion.version_parent to refer to viewonly relationship Person.versions will be deprecated in SQLAlchemy 1.4, and will be disallowed in a future release.  viewonly relationships should not be mutated (this warning may be suppressed after 10 occurrences)
sqlalchemy/orm/relationships.py:1912: SAWarning: Setting backref / back_populates on relationship Person.versions to refer to viewonly relationship PersonVersion.version_parent will be deprecated in SQLAlchemy 1.4, and will be disallowed in a future release.  viewonly relationships should not be mutated (this warning may be suppressed after 10 occurrences)
sqlalchemy/orm/relationships.py:1912: SAWarning: Setting backref / back_populates on relationship BillVersion.version_parent to refer to viewonly relationship Bill.versions will be deprecated in SQLAlchemy 1.4, and will be disallowed in a future release.  viewonly relationships should not be mutated (this warning may be suppressed after 10 occurrences)
sqlalchemy/orm/relationships.py:1912: SAWarning: Setting backref / back_populates on relationship Bill.versions to refer to viewonly relationship BillVersion.version_parent will be deprecated in SQLAlchemy 1.4, and will be disallowed in a future release.  viewonly relationships should not be mutated (this warning may be suppressed after 10 occurrences)
```

This is with `SQLAlchemy==1.3.16`

Interestingly, the warning says that this will be deprecated in SQLAlchemy 1.4 (but it already complains about it so it's already kind of deprecated in 1.3)

**Comment by Glandos** [comment]
This was introduced by SQLAlchemy-Continuum. This should be fixed in 1.3.10 by https://github.com/kvesteri/sqlalchemy-continuum/pull/240 but I am trying to figure out why this is not installed by our setup.cfg.

**Comment by Glandos** [comment]
Nevermind: this is just my virtualenv that was too old.
